### PR TITLE
Add Devices directory with stub of LiquidCrystal_TC

### PR DESCRIPTION
**Pull request by jgfoster**
We will be creating wrappers for the external libraries and this provides an initial stub.

### CODE REVIEW

File: examples/Blink/Blink.ino [hunk 1]
```diff
@@ -1,10 +1,10 @@
```
Comment (prestoncarman) [inline examples/Blink/Blink.ino]: I think we should rename this file to the something more appropriate. TankController?

File: .github/workflows/Arduino-CI.yaml [hunk 2]
```diff
@@ -20,6 +20,12 @@ jobs:
       - run: |
           bundle install
 
+      # Creates the Library directory and install our libraries
+      - run: |
+          bundle exec ensure_arduino_installation.rb
```
Comment (prestoncarman) [inline .github/workflows/Arduino-CI.yaml]: Is this needed for running tests locally or only for the CI?

File: examples/Blink/Blink.ino (hunk 1 above)
Comment (jgfoster) [reply examples/Blink/Blink.ino]: Yes, done.

File: .github/workflows/Arduino-CI.yaml (hunk 2 above)
Comment (jgfoster) [reply .github/workflows/Arduino-CI.yaml]: Yes, we need to install custom libraries so it does make sense to have an install script that is called by the testing framework and is the one we can use as well.

**Review by prestoncarman — APPROVED** [review]
LGTM

**Pull request merged**